Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2_description.md

```diff
@@ -7,3 +7,6 @@ In case the TAXII 2 server you're trying to connect to requires a custom authent
 
```

Comment:
Lines 1 and 5 should be all capitalized: Using API Token Authentication, Using Custom Authentication Header

Reply:
This is against the guideline that was given to us to use Sentence case and not Title Case.